platform/positionmanager/entrypoint.py

```diff
@@ -88,8 +90,64 @@ def get_predictions() -> dict[str, any]:
     return None
 
 
-if __name__ == "__main__":
+async def listener(consumer, producer, output_topic):
+    while True:
+        try:
+            async for message in consumer:
+                logger.info(f"Message received: {message}")
+                get_predictions()
+                await producer.send_and_wait(output_topic.name, b"test")
+                logger.info("Processed message and sent result")
+        except Exception as e:
+            await producer.send_and_wait(output_topic.name.replace("success", "error"), b"BROKEN!")
+            logger.error(f"Error in listener: {e!s}")
+
+
+async def main():
+    loop = asyncio.get_event_loop()
+
+    topic = Topic(environment="paper",
+                  fund="pocketsizefund",
+                  domain="trade",
+                  event="psf.cron.submitted",
+                  group_id="psf.cron")
+    output_topic = Topic(environment="paper",
```

Comment:
We should read in the `environment` from the environment or otherwise reference it dynamically.

Reply:
getting rid of environment altogether, will be implicit in the k8s namespace/kafka server